.github/workflows/main.yml

```diff
@@ -57,3 +56,30 @@ jobs:
         authkey: ${{ secrets.TAILSCALE_AUTHKEY }}
         version: 1.22.2
     - run: cd integration-tests && npm run test:${{ matrix.protocol }} -- --maxWorkers=8
+
+  integration-tests-flextesa:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 12.x
+    - run: npm ci
+    - run: npm run lerna -- bootstrap
```

Comment:
We should use the npm interface everywhere, so that changes to how it works are centralized

Reply:
(Does this environment use package.json?)

Reply:
Are you asking for a change in the doc?